File: BeatSync-Service-Persistence/src/main/java/com/astimefades/beatsyncservice/model/request/LoginAccountRequest.java
package com.astimefades.beatsyncservice.model.request;

import com.fasterxml.jackson.annotation.JsonProperty;

public class LoginAccountRequest {

    private String email;
    private String password;

    @JsonProperty("email")
    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    @JsonProperty("password")
    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}

File: BeatSync-Client-Android/app/src/main/java/com/astimefades/beatsyncandroid/model/request/LoginAccountRequest.kt
package com.astimefades.beatsyncandroid.model.request

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * Created by tobycaulk on 3/2/18.
 */
data class LoginAccountRequest(@JsonProperty("email") val email: String,
                               @JsonProperty("password") val password: String)